reference/forms/types/options/choice_name.rst.inc

```diff
@@ -9,3 +9,8 @@ of the choice views in the template.
 
 This can be a callable or a property path. See `choice_label`_ for similar usage.
 If ``null`` is used, an incrementing integer is used as the name.
+
+caution::
+
+    The configured value must be a valid form name. Make sure to only return
```

Comment:
Do we have a link to some article that explains what is a "valid form name" ?  Or can we explain what's that in a short sentence? Thanks.

Reply:
I have pushed an update. What do you think about the changed text?

Reply:
Now it's perfectly explained! Thanks.